wallet-new/README.md

```diff
@@ -73,13 +73,35 @@ $ stack exec cardano-node -- --topology=wallet-new/topology-examples/testnet.yam
 
 From there, you can browse the API documentation for V0 and V1 through the following URLs:
 
+<<<<<<< HEAD
```

Comment:
@jcmincke Argh, he it goes another conflict.

I would use the version of the readme shipped with `1.2.0` (so let's remove here everything from <<<HEAD to ====)